### ISSUE TITLE
cpplint installed but not working on vscode

### ISSUE THREAD

**Issue by TeachMeTW** [secularbird/cpplint-extension#42]
![1](https://user-images.githubusercontent.com/80879010/123032283-8b465a00-d39a-11eb-88a7-8ba8ee23168b.PNG)
![2](https://user-images.githubusercontent.com/80879010/123032284-8bdef080-d39a-11eb-89df-117b84c06b1b.PNG)

![4](https://user-images.githubusercontent.com/80879010/123032286-8c778700-d39a-11eb-90b9-178ad717c661.PNG)
![5 1](https://user-images.githubusercontent.com/80879010/123032288-8c778700-d39a-11eb-8f57-19cdbaca7bc8.PNG)
Basically, cpplint is installed but it is not working on the vscode gui. I can run the command and it shows the problems but does not highlight or do anything to vscode.

**Comment by weycen** [comment]
i'm win10, use python3.9 inatall the cpplint.exe, not working on vscode.  
I found the problem, the folder or file opened by VSCode cannot contain Chinese characters...

**Comment by BIAOXYZ** [comment]
I met a similar issue:
- First, I am sure cpplint is installed
```sh
$ cpplint

Syntax: cpplint.py [--verbose=#] [--output=vs7] [--filter=-x,+y,...]
                   [--counting=total|toplevel|detailed] [--root=subdir]
                   [--linelength=digits]
        <file> [file] ...

  The style guidelines this tries to follow are those in
    http://google-styleguide.googlecode.com/svn/trunk/cppguide.xml

  Every problem is given a confidence score from 1-5, with 5 meaning we are
  certain of the problem, and 1 meaning it could be a legitimate construct.
  This will miss some errors, and is not a substitute for a code review.

  To suppress false-positive errors of a certain category, add a
  'NOLINT(category)' comment to the line.  NOLINT or NOLINT(*)
  suppresses errors of all categories on that line.

  The files passed in will be linted; at least one file must be provided.
  Default linted extensions are .cc, .cpp, .cu, .cuh and .h.  Change the
  extensions with the --extensions flag.

  Flags:

    output=vs7
      By default, the output is formatted to ease emacs parsing.  Visual Studio
      compatible output (vs7) may also be used.  Other formats are unsupported.
```
- Second, it is also correctly configured in vscode's `settings.json`.
![image](https://user-images.githubusercontent.com/6740021/128183696-1e93a5b4-4b53-41a4-8b26-8989ad9046cb.png)

However, it never displays code-style or other info in vscode...

**Comment by weycen** [comment]
@BIAOXYZ 
Is your operating system Windows? If it is, i think your path is wrong, here are my settings:
```
"cpplint.cpplintPath": "D:\\ProgramFiles\\Python39\\Scripts\\cpplint.exe"
```
Maybe you can  wrapper cpplint.py as a script cpplint.bat, refer to #16 . I have tried this method, it is feasible.

**Comment by BIAOXYZ** [comment]
> @BIAOXYZ
> Is your operating system Windows? If it is, i think your path is wrong, here are my settings:
> 
> ```
> "cpplint.cpplintPath": "D:\\ProgramFiles\\Python39\\Scripts\\cpplint.exe"
> ```
> 
> Maybe you can wrapper cpplint.py as a script cpplint.bat, refer to #16 . I have tried this method, it is feasible.

@weycen Thanks a lot for your help.

The OS of my development machine (where the project as well as the `settings.json` file lies in) is Debian 9. However, I use vscode's remote ssh mode, so the picture is a screenshot from a Mac Book Pro.

I've already tried adding `.py` to the end, namely, using this (`"/usr/local/bin/cpplint.py"`) as the value of key `"cpplint.cpplintPath"`. But vscode immediately prompt this:

![image](https://user-images.githubusercontent.com/6740021/128369217-87ce7e33-1b77-4e66-a096-9380f80a7cd9.png)

**Comment by SimonNie98** [comment]
> > @BIAOXYZ
> > Is your operating system Windows? If it is, i think your path is wrong, here are my settings:
> > ```
> > "cpplint.cpplintPath": "D:\\ProgramFiles\\Python39\\Scripts\\cpplint.exe"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Maybe you can wrapper cpplint.py as a script cpplint.bat, refer to #16 . I have tried this method, it is feasible.
> 
> @weycen Thanks a lot for your help.
> 
> The OS of my development machine (where the project as well as the `settings.json` file lies in) is Debian 9. However, I use vscode's remote ssh mode, so the picture is a screenshot from a Mac Book Pro.
> 
> I've already tried adding `.py` to the end, namely, using this (`"/usr/local/bin/cpplint.py"`) as the value of key `"cpplint.cpplintPath"`. But vscode immediately prompt this:
> 
> ![image](https://user-images.githubusercontent.com/6740021/128369217-87ce7e33-1b77-4e66-a096-9380f80a7cd9.png)

Same problem here. Is there any solution?
